packages/app/src/sections/dashboard/Stake/EscrowTable.tsx

```diff
@@ -79,43 +90,96 @@ const EscrowTable = () => {
 	const openConfirmModal = useCallback(() => setConfirmModalOpen(true), [])
 	const closeConfirmModal = useCallback(() => setConfirmModalOpen(false), [])
 
+	const EscrowStatsContainer = () => (
+		<FlexDivRowCentered columnGap="25px" justifyContent="flex-end">
+			<FlexDivCol alignItems="flex-end">
+				<Body color="secondary">{t('dashboard.stake.tabs.escrow.total')}</Body>
+				<Body color="primary">
+					{truncateNumbers(totalVestable, 4)} {t('dashboard.stake.tabs.stake-table.kwenta-token')}
+				</Body>
+			</FlexDivCol>
+			<FlexDivCol alignItems="flex-end">
+				<Body color="secondary">{t('dashboard.stake.tabs.escrow.fee')}</Body>
+				<Body color="primary">
+					{truncateNumbers(totalFee, 4)} {t('dashboard.stake.tabs.stake-table.kwenta-token')}
+				</Body>
+			</FlexDivCol>
+			<Button
+				variant="yellow"
+				size="xsmall"
+				isRounded
+				disabled={!vestEnabled}
+				onClick={openConfirmModal}
+			>
+				{t('dashboard.stake.tabs.escrow.vest')}
+			</Button>
+		</FlexDivRowCentered>
+	)
+
 	return (
 		<EscrowTableContainer $noPadding>
-			<DesktopOnlyView>
+			<LargeScreenView>
 				{/*@ts-expect-error*/}
 				<StyledTable
 					data={escrowData}
 					compactPagination
-					pageSize={10}
+					customizePagination={true}
+					pageSize={4}
 					showPagination
 					columnsDeps={columnsDeps}
+					children={<EscrowStatsContainer />}
 					columns={[
 						{
-							Header: () => <input type="checkbox" checked={checkAllState} onChange={selectAll} />,
+							Header: () => (
+								<Checkbox
+									id="header"
+									label=""
+									checked={checkAllState}
+									onChange={selectAll}
+									variant="table"
+									checkSide="right"
+								/>
+							),
 							Cell: (cellProps: CellProps<EscrowData>) => (
-								<input
+								<Checkbox
+									id={cellProps.row.index.toString()}
 									key={cellProps.row.index}
-									type="checkbox"
 									checked={checkedState[cellProps.row.index]}
 									onChange={handleOnChange(cellProps.row.index)}
```

Comment:
should this be `() => handleOnChange(cellProps.row.index)`?

Reply:
Yes. Fixed.